flutter/flutter.mk

```diff
@@ -65,9 +65,9 @@ flutter_folder_args=${flutter_data_folder_arg} ${flutter_cache_folder_arg}
 
 FIREBASE_ENV_FILE?=flutter/lib/firebase/firebase_options.env
 -include ${FIREBASE_ENV_FILE}
-export
 .PHONY: flutter/firebase-config
 flutter/firebase-config:
+	export
```

Comment:
Why there is this change?

Reply:
The export command on line 68 previously somehow impacts building the windows app in powershell
so, i had to move this to inside the task